crates/rpc/src/jsonrpc/router/subscription.rs

```diff
@@ -855,12 +855,12 @@ mod tests {
             .await
             .unwrap();
         let res = sender_rx.recv().await.unwrap().unwrap();
-        let subscription_id = match res {
+        let subscription_id: u64 = match res {
             Message::Text(json) => {
                 let json: serde_json::Value = serde_json::from_str(&json).unwrap();
                 assert_eq!(json["jsonrpc"], "2.0");
                 assert_eq!(json["id"], 1);
-                json["result"].as_u64().unwrap()
+                json["result"].as_str().unwrap().parse().unwrap()
```

Comment:
I suppose there's no reason to change it in this one place, but do we generally just crash on unexpected inputs?

Reply:
No. `unwrap()` / `expect()` used here is allowed only because this is a test.